katello/hooks/pre_commit/14-cdn_setting.rb

```diff
@@ -0,0 +1,12 @@
+if module_enabled?('katello')
+  # Using @kafo.config.answers because it will have the answers prior to being
+  # committed to disk and being overwritten
+  # https://github.com/theforeman/foreman-installer/pull/477#pullrequestreview-370785117
+  param = @kafo.config.answers['katello']['cdn_ssl_version']
```

Comment:
`param` is a method that exists in the hook context and it's not a good practice to overwrite this. I think `value` is a better name.